src/icons/globe.jsx

```diff
@@ -0,0 +1,15 @@
+export default ({ width, height }) => {
+  return (
+    <svg
+      width={width || '25'}
```

Comment:
Should this be 24 as well? Viewbox is square

Reply:
Done